interface/usergroup/facilities_add.php

```diff
@@ -184,7 +184,7 @@ function displayAlert() {
             <input type="hidden" name="mode" value="facility" />
 
             <div class="row">
-                <div class="col-sm-6">
+                <div class="col-6">
```

Comment:
Not sure why we need to restrict to col-6 inside a dialog. Especially for a vertical form. Forms in dialogs should be 100% width where possible as they tend to behave better in responsive and will sixe to the calling dialog ie looks like a lot of padding. This keeps from sizing the dialog to the form. IMO, mind you.
Just leave though for now and ketch on BS alert issue which I think you'll find fun to do:)